tools/renovate/sdk.json

```
@@ -1,87 +1,10 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "description": "Group all otter dependencies and trigger SDK regeneration",
-  "labels": [
-    "dependencies"
-  ],
-  "updateInternalDeps": true,
-  "automerge": true,
-  "platformAutomerge": true,
-  "packageRules": [
-    {
-      "groupName": "Otter dependencies",
-      "groupSlug": "otter-dependencies",
-      "matchPackagePrefixes": [
-        "@otter",
-        "@o3r",
-        "@ama-sdk",
-        "@ama-terasu"
-      ]
-    },
-    {
-      "matchBaseBranches": [
-        "master",
-        "main"
-      ],
-      "matchPackagePrefixes": [
-        "@ama-sdk"
-      ],
-      "rangeStrategy": "bump"
-    },
-    {
-      "matchBaseBranches": [
-        "/^release/.*/"
-      ],
-      "matchPackagePrefixes": [
-        "@ama-sdk"
-      ],
-      "rangeStrategy": "patch"
-    },
-    {
-      "matchBaseBranches": [
-        "master",
-        "main",
-        "/^release/.*/"
-      ],
-      "matchPackagePrefixes": [
-        "@ama-sdk"
-      ],
-      "matchPackageNames": [
-        "npm"
-      ],
-      "postUpgradeTasks": {
-        "commands": [
-          "npm install",
-          "npm run swagger:generate"
-        ],
-        "fileFilters": [
-          "!**/.{npmrc,yarnrc*}"
-        ],
-        "executionMode": "branch"
-      }
-    },
-    {
-      "matchBaseBranches": [
-        "master",
-        "main",
-        "/^release/.*/"
-      ],
-      "matchPackagePrefixes": [
-        "@ama-sdk"
-      ],
-      "matchPackageNames": [
-        "yarn"
-      ],
-      "postUpgradeTasks": {
-        "commands": [
-          "yarn install",
-          "yarn run swagger:generate"
-        ],
-        "fileFilters": [
-          "!**/.{npmrc,yarnrc*}"
-        ],
-        "executionMode": "branch"
-      }
-    }
+  "description": "Group all otter dependencies and trigger SDK regeneration with yarn",
+  "extends": [
+    "github>AmadeusITGroup/otter//tools/renovate/group/otter",
+    "github>AmadeusITGroup/otter//tools/renovate/tasks/base",
+    "github>AmadeusITGroup/otter//tools/renovate/tasks/sdk-regeneration(yarn)",
```

This preset does not exist

```suggestion
    "github>AmadeusITGroup/otter//tools/renovate/tasks/sdk-regenerate(yarn)",
```
?